control-plane-operator/controllers/hostedcontrolplane/oauth/deployment.go

```diff
@@ -134,13 +170,24 @@ func oauthContainerMain() *corev1.Container {
 	}
 }
 
-func buildOAuthContainerMain(image string, noProxy []string) func(c *corev1.Container) {
+func buildOAuthContainerMain(image string, auditWebhookRef *corev1.LocalObjectReference, noProxy []string) func(c *corev1.Container) {
 	return func(c *corev1.Container) {
 		c.Image = image
 		c.Args = []string{
 			"osinserver",
 			fmt.Sprintf("--config=%s", path.Join(volumeMounts.Path(c.Name, oauthVolumeConfig().Name), OAuthServerConfigKey)),
+			"--audit-log-format=json",
```

Comment:
Do we have provenance for these specific flags?

Reply:
They are used in the oauth-openshift repo here: https://github.com/openshift/cluster-authentication-operator/blob/b415439ebab2829c8da1ea17c05f2ac75fe5dbe8/bindata/oauth-apiserver/deploy.yaml#L68

And I also checked a real oauth-openshift pod in the standalone OCP
```
$ oc get deploy -n openshift-authentication oauth-openshift -oyaml
...
 - args:
        - |
          if [ -s /var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle/ca-bundle.crt ]; then
              echo "Copying system trust bundle"
              cp -f /var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle/ca-bundle.crt /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
          fi
          exec oauth-server osinserver \
          --config=/var/config/system/configmaps/v4-0-config-system-cliconfig/v4-0-config-system-cliconfig \
          --v=2 \
          --audit-log-format=json \
          --audit-log-maxbackup=10 \
          --audit-log-maxsize=100 \
          --audit-log-path=/var/log/oauth-server/audit.log \
          --audit-policy-file=/var/run/configmaps/audit/audit.yaml
        command:
        - /bin/bash
        - -ec
        image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:9c5863100af97f244bc0af6c727785a7487eb8514ee549a41089bf6b06315e0f
...
        name: oauth-openshift
```

The only difference is `--audit-log-maxbackup=1`. Because in hypershift codes, it is set to 1 for the existing audit components (kas, oapi).

Reply:
Awesome, thanks for the explanation!

Reply:
It'd be nice to drop the link in a code comment so everyone gets the question before the answer when reading the code.